CHANGELOG.rst

```diff
@@ -6,6 +6,9 @@ Changelog
 
 * Add low level bindings for extracting the seed and the public key
   from crypto_sign_ed25519 secret key
+* Cancel hypothesis too_slow health check for test_crypto_kx_seed_keypair_
+  seed_too_large to fix build on very slow architectures such mips. GitHub
```

Comment:
Please, put a newline before the `test_crypto...` function name, and join the two parts, to restore correct docs building.